atc/db/resource_test.go

```diff
@@ -195,7 +195,6 @@ var _ = Describe("Resource", func() {
 					Expect(buildSummary.Name).To(Equal(db.CheckBuildName))
 					Expect(buildSummary.TeamName).To(Equal(scenario.Team.Name()))
 					Expect(buildSummary.PipelineName).To(Equal(scenario.Pipeline.Name()))
-					Expect(buildSummary.Status).To(Equal(atc.StatusFailed))
```

Comment:
Should we be testing it to return the status that is in the db? (Pending build?)

Reply:
There is no such status in db since the build (id 99) never exists. The status in the removed line comes from 
https://github.com/concourse/concourse/blob/a188c2a9d671597cd3f724807800731a6df563be/atc/db/resource.go#L1003

and 

https://github.com/concourse/concourse/blob/a188c2a9d671597cd3f724807800731a6df563be/atc/db/resource.go#L133

since in the test when initializing the resource config scope the field never got set to true. And there is no proper way to set it othan than DB surgery..